src/main/java/edu/unc/lib/boxc/migration/cdm/InitializeProjectCommand.java

```diff
@@ -87,48 +90,60 @@ public Integer call() throws Exception {
             log.error("Unable to read application configuration", e);
             return 1;
         }
-        var cdmEnvConfig = config.getCdmEnvironments().get(cdmEnvId);
 
         Path currentPath = parentCommand.getWorkingDirectory();
         String projDisplayName = projectName == null ? currentPath.getFileName().toString() : projectName;
         String collId = cdmCollectionId == null ? projDisplayName : cdmCollectionId;
+        String username = System.getProperty("user.name");
 
-        // Retrieve field information from CDM
-        CdmFieldInfo fieldInfo;
-        try {
-            fieldService.setCdmBaseUri(cdmEnvConfig.getHttpBaseUrl());
-            fieldInfo = fieldService.retrieveFieldsForCollection(collId);
-        } catch (IOException | MigrationException e) {
-            log.error("Failed to retrieve field information for collection in project", e);
-            outputLogger.info("Failed to retrieve field information for collection {} in project {}:\n{}",
-                    collId, projDisplayName, e.getMessage());
-            return 1;
-        }
+        if (newProject != null && !newProject.isBlank()) {
+            // Instantiate the project
+            try {
+                MigrationProjectFactory.createMigrationProject(
```

Comment:
It would probably also make sense to store the value of this property into the project properties file, in case we need to check the value in later steps to see if the user is allowed to do them, or if chompb needs to pull data from a particularly location, etc. We could have separate `MigrationProjectFactory.createCdmMigrationProject` and `MigrationProjectFactory.createFilesMigrationProject` (they could both call MigrationProjectFactory.createMigrationProject internally) which would allow more control over what params get passed in. Or just keep one method and pass the attribute in as another param.